wp-content/themes/core/assets/css/src/theme/forms/_variables.pcss

```diff
@@ -5,68 +5,45 @@
  * ----------------------------------------------------------------------------- */
 
 :root {
-	/* -----------------------------------------------------------------------------
-	 * Borders
-	 * ----------------------------------------------------------------------------- */
-
-	--form-border-color: lightgrey;
-	--form-border-color-focus: darken(var(--form-border-color), 0.25);
-	--form-border-size: 1px;
-	--form-border-size-focus: var(--form-border-size);
-	--form-border-radius: 0;
 
 	/* -----------------------------------------------------------------------------
 	 * Colors
 	 * ----------------------------------------------------------------------------- */
 
 	--form-color: var(--color-text);
-	--form-color-secondary: var(--color-grey-light);
+	--form-color-secondary: var(--color-neutral-50);
 	--form-color-error: red;
 	--form-color-required: var(--form-color-error);
 	--form-color-success: green;
-	--form-color-placeholder: grey;
-
-	/* -----------------------------------------------------------------------------
-	 * UI
-	 * ----------------------------------------------------------------------------- */
-
-	--form-ui-icon-color: lightgrey; /* Control icons */
-	--form-ui-icon-font-size: 20px; /* Control icons */
-	--form-ui-color: var(--color-grey-dark); /* Select icon */
-	--form-ui-bgd-color: var(--form-border-color);  /* Select, Checkbox, & Radio */
-	--form-ui-bgd-color-focus: var(--form-border-color-focus);  /* Select */
-	--form-ui-icon-select: '\e900'; /* Select */
-	--form-ui-icon-select-font-size: 25px; /* Select */
```

Comment:
In going through and scoping out the clean up, as well as some of the lingering issues from my first pass, I'm noticing that there is a global set of UI based vars that would be nice to have/use over the more granular vars, a great example is sticking with `--form-ui-bgd-color` vs. `--form-control-bgd-color` and cleaning up the latter. I would imagine we'll have a whole global set of stuff like this that would be lovely to standardize into a set of UI based vars.

Reply:
@ryanurban @dpellenwood Let's chat through this. I feel that the global UI vars are too general and it's hard to recall where each is used. For me, this is an area that contributes to the "nesting" of forms vars that makes it tedious to update on project setup. Many in this set were only used in 1-2 spots.